providers/mlx5/mlx5.h

```diff
@@ -90,10 +91,12 @@ enum {
 	MLX5_DBG_CQ_CQE		= 1 << 4,
 	MLX5_DBG_CONTIG		= 1 << 5,
 	MLX5_DBG_DR		= 1 << 6,
+	MLX5_DBG_VFIO		= 1 << 7,
```

Comment:
As I said, I don't want to see any change here and I agree with Gal, this mask is better to be deleted.

IMHO, this mask represents misunderstanding on how libraries are distributed/installed/used in the real world. And if someone wants to debug library, he/she needs to dump everything to the file and use editor later.

Thanks

Reply:
OK, this extra bit mask / usage was dropped, a rebased version was uploaded.